### PR TITLE
completed module_8

### PR DESCRIPTION
**Pull request by vladSHOKO**
@Lehersun module 8 ready to check

conditions in files

### CODE REVIEW

File: module_8/002_strings_function.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+$url = '/some/path/to/page/';
+var_dump(trim($url, "/"));
+
+
+
+
+
+// 2. Уберите из строки все запятые и точки, а пробелы замените на тире
+$line = 'Good news, everyone.';
+$withoutDots = str_replace(',', '', $line );
```
Comment (Lehersun) [inline module_8/002_strings_function.php]: code style

File: module_8/002_strings_function.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+
+// 4. Найдите на какой позиции находится запятая в строке $line и выведите это значение
+$position = strpos($line, ',');
+var_dump($position);
+
+
+// 5. Вырежьте из строки $line подстроку после запятой до предпоследнего символа с конца
+var_dump(substr($line, strpos($line, ',')));
+
```
Comment (Lehersun) [inline module_8/002_strings_function.php]: Read task carefully

File: module_8/003_arrays_function.php [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+$hasVasyaPupkin = in_array('vasya@example.com', $emails);
+var_dump($hasVasyaPupkin);
+
+
+// 5. Используя подходящую встроенную функцию, добавьте в библиотеку еще одну книгу Мартина Фаулера
+$newBookOfFauler = [
+    'title' => 'Новая книга Фаулера',
+    'author' => 'martin_fauler@example.com',
+    'year' => 2010,
+    ];
```
Comment (Lehersun) [inline module_8/003_arrays_function.php]: Code style

File: module_8/003_arrays_function.php [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+array_push($library['books'], $newBookOfFauler);
+var_dump($library);
+
+
+// 6. Используя подходящую встроенную функцию, добавьте в библиотеку на первую полку еще одну книгу Мартина Фаулера
+$otherNewBookOfFauler = [
+    'title' => 'Ещё одна новая книга Фаулера',
+    'author' => 'martin_fauler@example.com',
+    'year' => 2010,
+    ];
```
Comment (Lehersun) [inline module_8/003_arrays_function.php]: Code style

**Review by Lehersun — CHANGES_REQUESTED** [review]

**Review by vladSHOKO — COMMENTED** [review]
@Lehersun code style remade

File: module_8/002_strings_function.php (hunk 2 above)
Comment (Lehersun) [reply module_8/002_strings_function.php]: @vladSHOKO, please read this task one more time and fix solution

**Review by vladSHOKO — COMMENTED** [review]
@Lehersun last exercise ready to check

File: module_8/002_strings_function.php (hunk 2 above)
Comment (vladSHOKO) [reply module_8/002_strings_function.php]: @Lehersun mistakes correct

Comment (Lehersun) [reply module_8/002_strings_function.php]: Good job

**Review by Lehersun — APPROVED** [review]

**Pull request merged**